src/env_loader.zig

```diff
@@ -1011,8 +999,9 @@ const Parser = struct {
         map: *Map,
         comptime override: bool,
         comptime is_process: bool,
-        comptime conditional: bool,
     ) void {
+        Analytics.Features.dotenv = true;
```

Comment:
this should only be true if it's replacing the dotenv package

Reply:
this flag was set at every point (like 10 of them)
```
   if (dir.hasComptimeQuery(".env.development")) {
      try this.loadEnvFile(dir_handle, ".env.development", false, true);
-     Analytics.Features.dotenv = true;
   }
```
I move this to the common root that all usages call into.

this was placed at the start of the function so that a crash in dotenv parser would be marked as using dotenv

Reply:
I see, makes sense